netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -362,6 +364,13 @@ protected NettyServer buildTransportServer(
       negotiator = sslContext != null ? ProtocolNegotiators.serverTls(sslContext) :
               ProtocolNegotiators.serverPlaintext();
     }
+
+    if (maxConnectionAgeInNanos != MAX_CONNECTION_AGE_NANOS_DISABLED) {
+      // apply a random jitter of +/-10% to max connection age
```

Comment:
The intention was that this would be per-connection. That wasn't specified well. I've updated the spec doc.